src/Schema/MySQLSchemaManager.php

```diff
@@ -211,6 +212,10 @@ protected function _getPortableTableColumnDefinition(array $tableColumn): Column
             'autoincrement' => str_contains($tableColumn['extra'], 'auto_increment'),
         ];
 
+        if ($dbType === 'enum' && preg_match_all("/'([^']+)'/", $tableColumn['type'], $members) !== false) {
```

Comment:
`preg_match_all()` returns `false` only if an error occurred, for instance if the regular expression is invalid. I'm pretty sure that's not the case you wanted to catch here.